features/api.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Client behaviour for the API endpoints
+
+    @series.all
+    @uses.config.machine_type.lxd.container
+    Scenario Outline: API invalid endpoint or args
```

Comment:
The only problem with the warning test is that I don't call directly any of the endpoints here, except initiate.
If we use initiate for that, we would need to create distinct steps for the warning situation because we should also revoke the token at the end. So I am still unsure if it is worth it, but let me know what you all think

Reply:
Well it is covered by unit tests anyway, we can add it later to behave using a simpler scenario. Leave it on me.